docs/modules/ROOT/pages/index.adoc

```diff
@@ -478,6 +478,9 @@ Update this script in package.json:
 [#vite]
 === Vite
 Any app created with https://vitejs.dev/guide/[Vite] are compatible without any change and Hot Module Replacement (HMR) should work by default.
+[#solidjs]
+=== SolidJS
+Any app created with https://www.solidjs.com/[SolidJS] are compatible without any change and Hot Module Replacement (HMR) should work by default. It is based on link:#vite[Vite].
 
```

Comment:
I think change this doc to add a section that just says and the following application.properties must be set.

```properties
quarkus.quinoa.dev-server.port=3001
```